app/services/integration_service.rb

```diff
@@ -30,4 +30,8 @@ def call
       end
     end
   end
+
+  def self.vendor_module(integration)
+    "#{integration.vendor.camelize}Service::Batch".constantize
```

Comment:
Is it correct that the `Batch` completion handler is hardcoded here?

Reply:
I think this is redundant now. I'll confirm later and remove if so.

Reply:
Confirmed & removed.